### PR TITLE
Fix MSA e2e tests

### PR DESCRIPTION
**Pull request by hectormmg**
This PR:
- Replaces sign-in button ID with #kmsiTitle ID for KMSI page in MSA flows

### CODE REVIEW

File: samples/e2eTestUtils/src/TestUtils.ts [hunk 1]
```diff
@@ -307,7 +307,22 @@ export async function enterCredentials(
             throw e;
         });
     } catch (e) {
-        return;
+        // keep me signed in MSA
+        try {
+            await page.waitForSelector("#kmsiTitle", { timeout: 1000 });
```
Comment (tnorling) [inline samples/e2eTestUtils/src/TestUtils.ts]: Could be a bit faster and less duplication if you use Promise.any above with both tags

**Review by tnorling — APPROVED** [review]
One suggestion but non-blocking

File: samples/e2eTestUtils/src/TestUtils.ts (hunk 1 above)
Comment (hectormmg) [reply samples/e2eTestUtils/src/TestUtils.ts]: Great idea, thanks! I was wondering if there was a way to use whichever one showed up.

**Review by lalimasharda — APPROVED** [review]

File: samples/e2eTestUtils/src/TestUtils.ts (hunk 1 above)
Comment (hectormmg) [reply samples/e2eTestUtils/src/TestUtils.ts]: @tnorling ended up using race since .any wasn't available for the current build target settings.

**Review by konstantin-msft — APPROVED** [review]

**Pull request merged**